### ISSUE TITLE
Improve documentation for --cms-base-url="https://example.org"

### ISSUE THREAD

**Issue by jenlampton** [civicrm/cv#94]
I am following [the installation instructions](https://docs.civicrm.org/installation/en/latest/general/cli-cv/) for Civi and I'm stumped at how to build the command to install civi. I am a complete Civi Novice, but I think it will be exactly the right solutuon for one of my clients, so I really want to give it a try. The example provided is `cv core:install --cms-base-url="URL" --lang="LANG"`  But there are things in here I don't know how to replace -- let's look at `URL`.

The documentation says `The "CMS Base URL" is the public URL of the website. It is required for most CLI installations.`

So... that would mean I should put in the URL of the live website where Civi will eventually be deployed? If I enter the live URL when I'm installing Civi on a test site (where the URL is different) would Civi still work? Or does it need to match the current url of the site (similar to Drupal/Backdrop), and if that's the case, will I be able to change this later, when I launch the site?

(the above was my complete thought process while trying to install civi -- maybe not the best format for this ticket, but I wanted to write it all down while it's fresh in my mind.)

Also, if there is documentation I should be consulting other than https://docs.civicrm.org/installation please let me know if I am looking in the wrong place :)

**Comment by jenlampton** [comment]
> Or does it need to match the current url of the site (similar to Drupal/Backdrop), and if that's the case, will I be able to change this later, when I launch the site?

I chose poorly. I guessed that I should use the current URL of the local site where civi is running on my laptop, but now every time I submit any form in civi, I get the error:

```
Sorry, due to an error, we are unable to fulfill your request at the moment. You may want to contact your administrator or service provider with more details about what action you were performing when this occurred.
We can't load the requested web page. This page requires cookies to be enabled in your browser settings. Please check this setting and enable cookies (if they are not enabled). Then try again. If this error persists, contact the site administrator for assistance.

Site Administrators: This error may indicate that users are accessing this page using a domain or URL other than the configured Base URL. EXAMPLE: Base URL is http://example.org, but some users are accessing the page via http://www.example.org or a domain alias like http://myotherexample.org.

Error type: Could not find a valid session key.

Return to home page.
```

**Comment by MegaphoneJon** [comment]
The local site URL is the correct URL.  That error suggests you may have selected HTTP instead of HTTPS or vice versa though.

**Comment by jenlampton** [comment]
I checked that, both are `http`. (I am checking the value that was automatically entered into the `civicrm.settings.php` file, I think that's the right place?)

**Comment by MegaphoneJon** [comment]
It is, it should be the `CIVICRM_UF_BASEURL` constant in `civicrm.settings.php`.

**Comment by jenlampton** [comment]
Yeah, that matches what I see in my browser URL bar (http and everything). I'm still convinced I may not have installed civi correctly, maybe I'll throw it out and start over again. I'll try to use the UI this time and see if I get different results.
